sources/packages/backend/apps/api/src/utilities/datatable-config.ts

```diff
@@ -28,6 +32,14 @@ export interface ProgramPaginationOptions extends BasePaginationOptions {
   inactiveProgramSearch?: boolean;
 }
 
+/**
+ * CAS invoices specific parameters.
+ */
+export interface CASInvoicePaginationOptionsAPIInDTO
```

Comment:
Please refrain from using the DTO convention outside controller scope. it must be called `CASInvoicePaginationOptions`